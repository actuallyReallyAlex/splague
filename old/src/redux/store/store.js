import { createStore, combineReducers } from 'redux'
import cureReducer from '../reducers/cure'
import logReducer from '../reducers/log'
import plagueReducer from '../reducers/plague'
import playerReducer from '../reducers/player'
import uiReducer from '../reducers/ui'
import worldReducer from '../reducers/world'

const store = createStore(
  combineReducers({
    cure: cureReducer,
    log: logReducer,
    plague: plagueReducer,
    player: playerReducer,
    ui: uiReducer,
    world: worldReducer
  }),
  window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
)

export default store
