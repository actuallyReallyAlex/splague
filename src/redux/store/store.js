import { createStore, combineReducers } from 'redux'
import WorldReducer from '../reducers/world'

const store = createStore(
  combineReducers({
    world: WorldReducer
  }),
  window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
)

export default store
