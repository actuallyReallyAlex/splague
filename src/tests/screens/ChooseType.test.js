import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import { Provider } from 'react-redux'
import store from '../../redux/store/store'
import ChooseType from '../../screens/ChooseType'

describe('<ChooseType />', () => {
  it('Should render snapshot of ChooseType screen.', () => {
    const component = renderer
      .create(
        <Provider store={store}>
          <ChooseType />
        </Provider>
      )
      .toJSON()
    expect(component).toMatchSnapshot()
  })
})
