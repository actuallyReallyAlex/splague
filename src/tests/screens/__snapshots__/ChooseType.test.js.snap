// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ChooseType /> Should render snapshot of ChooseType screen. 1`] = `
.c14 {
  display: inline-block;
  -webkit-flex: 0 0 auto;
  -ms-flex: 0 0 auto;
  flex: 0 0 auto;
  width: 24px;
  height: 24px;
  fill: #6FFFB0;
  stroke: #6FFFB0;
}

.c14 g {
  fill: inherit;
  stroke: inherit;
}

.c14 *:not([stroke])[fill="none"] {
  stroke-width: 0;
}

.c14 *[stroke*="#"],
.c14 *[STROKE*="#"] {
  stroke: inherit;
  fill: none;
}

.c14 *[fill-rule],
.c14 *[FILL-RULE],
.c14 *[fill*="#"],
.c14 *[FILL*="#"] {
  fill: inherit;
  stroke: none;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  outline: none;
  max-width: 100%;
  margin: 0px;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background: #252839;
  color: #f8f8f8;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  width: 100%;
  height: 100%;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  padding: 0px;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  outline: none;
  max-width: 100%;
  margin: 0px;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  padding: 0px;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  outline: none;
  max-width: 100%;
  margin-bottom: 12px;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  padding: 0px;
}

.c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  outline: none;
  max-width: 100%;
  margin-left: 12px;
  margin-right: 12px;
  margin-top: 6px;
  margin-bottom: 6px;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  padding: 0px;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  outline: none;
  max-width: 100%;
  margin: 0px;
  border-bottom: solid 1px rgba(255,255,255,0.33);
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  padding: 0px;
}

.c8 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  outline: none;
  max-width: 100%;
  margin: 0px;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  padding: 0px;
}

.c9 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  outline: none;
  margin: 0px;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-flex: 1 1;
  -ms-flex: 1 1;
  flex: 1 1;
  -webkit-flex-basis: auto;
  -ms-flex-preferred-size: auto;
  flex-basis: auto;
  padding: 0px;
}

.c13 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  box-sizing: border-box;
  outline: none;
  max-width: 100%;
  margin-left: 12px;
  margin-right: 12px;
  min-width: 0;
  min-height: 0;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-flex: 0 0 auto;
  -ms-flex: 0 0 auto;
  flex: 0 0 auto;
  padding: 0px;
}

.c7 {
  display: inline-block;
  box-sizing: border-box;
  cursor: pointer;
  outline: none;
  font: inherit;
  -webkit-text-decoration: none;
  text-decoration: none;
  margin: 0;
  background: transparent;
  overflow: visible;
  text-transform: none;
  color: inherit;
  border: none;
  padding: 0;
  text-align: inherit;
}

.c4 {
  font-size: 18px;
  line-height: 24px;
}

.c12 {
  box-sizing: border-box;
  font-size: inherit;
  border: none;
  -webkit-appearance: none;
  padding: 11px;
  outline: none;
  background: transparent;
  color: inherit;
  font-weight: 600;
  margin: 0;
  border: 1px solid rgba(255,255,255,0.33);
  border-radius: 4px;
  width: 100%;
  border: none;
}

.c12::-webkit-search-decoration {
  -webkit-appearance: none;
}

.c12::-webkit-input-placeholder {
  color: #AAAAAA;
}

.c12::-moz-placeholder {
  color: #AAAAAA;
}

.c12:-ms-input-placeholder {
  color: #AAAAAA;
}

.c12::-moz-focus-inner {
  border: none;
  outline: none;
}

.c10 {
  position: relative;
  width: 100%;
}

.c11 {
  cursor: pointer;
}

.c6 {
  border: 1px solid rgba(255,255,255,0.33);
  border-radius: 4px;
}

@media only screen and (max-width:768px) {
  .c0 {
    margin: 0px;
  }
}

@media only screen and (max-width:768px) {
  .c0 {
    padding: 0px;
  }
}

@media only screen and (max-width:768px) {
  .c1 {
    margin: 0px;
  }
}

@media only screen and (max-width:768px) {
  .c1 {
    padding: 0px;
  }
}

@media only screen and (max-width:768px) {
  .c2 {
    margin-bottom: 6px;
  }
}

@media only screen and (max-width:768px) {
  .c2 {
    padding: 0px;
  }
}

@media only screen and (max-width:768px) {
  .c3 {
    margin-left: 6px;
    margin-right: 6px;
  }
}

@media only screen and (max-width:768px) {
  .c3 {
    margin-top: 3px;
    margin-bottom: 3px;
  }
}

@media only screen and (max-width:768px) {
  .c3 {
    padding: 0px;
  }
}

@media only screen and (max-width:768px) {
  .c5 {
    margin: 0px;
  }
}

@media only screen and (max-width:768px) {
  .c5 {
    border-bottom: solid 1px rgba(255,255,255,0.33);
  }
}

@media only screen and (max-width:768px) {
  .c5 {
    padding: 0px;
  }
}

@media only screen and (max-width:768px) {
  .c8 {
    margin: 0px;
  }
}

@media only screen and (max-width:768px) {
  .c8 {
    padding: 0px;
  }
}

@media only screen and (max-width:768px) {
  .c9 {
    margin: 0px;
  }
}

@media only screen and (max-width:768px) {
  .c9 {
    padding: 0px;
  }
}

@media only screen and (max-width:768px) {
  .c13 {
    margin-left: 6px;
    margin-right: 6px;
  }
}

@media only screen and (max-width:768px) {
  .c13 {
    padding: 0px;
  }
}

<div
  className="animated fadeIn c0"
>
  <div
    className="c1"
  >
    <form
      onSubmit={[Function]}
    >
      <div
        className="c2"
      >
        <div
          className="c3"
        >
          <label
            className="c4"
          >
            Plague Type
          </label>
        </div>
        <div
          className="c5"
        >
          <button
            aria-label="Open Drop"
            className="c6 c7"
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            type="button"
          >
            <div
              className="c8"
            >
              <div
                className="c9"
              >
                <div
                  className="c10"
                >
                  <input
                    autoComplete="off"
                    className="c11 c12"
                    onBlur={[Function]}
                    onChange={[Function]}
                    onClick={[Function]}
                    onFocus={[Function]}
                    onKeyDown={[Function]}
                    readOnly={true}
                    tabIndex="-1"
                    type="text"
                    value=""
                  />
                </div>
              </div>
              <div
                className="c13"
                style={
                  Object {
                    "minWidth": "auto",
                  }
                }
              >
                <svg
                  aria-label="FormDown"
                  className="c14"
                  viewBox="0 0 24 24"
                >
                  <polyline
                    fill="none"
                    points="18 9 12 15 6 9"
                    stroke="#000"
                    strokeWidth="2"
                  />
                </svg>
              </div>
            </div>
          </button>
        </div>
      </div>
    </form>
  </div>
  
</div>
`;
